.github/PULL_REQUESTS/enhancement_template.md

```diff
@@ -0,0 +1,54 @@
+<!--
```

Comment:
See .github/CONTRIBUTING.md for which types of PRs we have (Branch names). This should probably renamed from enhancement -> feature.

Probably CI-related should get their own PR template, too?

Reply:
Good hint, thanks.